src/components/CreditCard/CreditCard.tsx

```diff
@@ -44,13 +41,15 @@ function CreditCard({
       <S.CreditCardICChip />
       <S.CreditCardInfoFooter>
         <S.CreditCardNumber>
-          {convertSecuredCreditCard(number).map((num, idx) => (
-            <div key={idx}>{num}</div>
-          ))}
+          {
+            convertSecuredCreditCard(number.length > 0 ? number : '0000000000000000').map((num, idx) => (
+              <div key={idx}>{num}</div>
+            ))
+          }
         </S.CreditCardNumber>
         <S.CreditCardContainer>
-          <S.CreditCardBox>{owner}</S.CreditCardBox>
-          <S.CreditCardBox>{markExpiry(expiry)}</S.CreditCardBox>
+          <S.CreditCardBox>{(owner !== undefined) && (owner !== '') ? owner : 'WOOWA COURSE'}</S.CreditCardBox>
```

Comment:
```suggestion
          <S.CreditCardBox>{owner || 'WOOWA COURSE'}</S.CreditCardBox>
```

요렇게 커버가 안되는 케이스가 있었나요?

Reply:
수정해봤는데 동일한 결과가 출력됩니다...! 제안해주신 코드가 훨씬 간결해보이네요